src/profile-logic/import/instruments/BinaryPlistParser.js

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Same for this class. It's probably fine to land it, but it would be so much better if we can find a well maintained npm package for this.